src/fides/api/models/location_regulation_selections.py

```diff
@@ -414,6 +414,17 @@ def _load_privacy_notice_regions() -> Dict[str, Union[Location, LocationGroup]]:
     {location.id: location.id for location in privacy_notice_regions_by_id.values()},
 )
 
+# Create a notice region enum that includes regions we no longer support but still preserve
+# on historical records for auditing purposes
+deprecated_gb_regions = ["gb_eng", "gb_sct", "gb_wls", "gb_nir"]
+current_regions = {
+    location.id: location.id for location in privacy_notice_regions_by_id.values()
+}
+current_regions.update({gb_region: gb_region for gb_region in deprecated_gb_regions})
+DeprecatedNoticeRegion: Enum = Enum(  # type: ignore[misc]
+    "DeprecatedNoticeRegion", current_regions
+)
+
```

Comment:
This is so you can access `PrivacyNoticeHistory.regions` in SQLAlchemy which still may have gb_eng regions - these were kept for auditing purposes